tests/phpunit/CRM/Consentactivity/UpgraderTest.php

```diff
@@ -30,7 +40,7 @@ public function testPostInstall()
             $this->assertEmpty($installer->install());
             $this->assertEmpty($installer->postInstall());
         } catch (Exception $e) {
-            $this->fail("Should not throw exception.");
+            $this->fail("Should not throw exception. ".$e->getMessage());
```

Comment:
I've seen you using this construct (`try..catch..$this->fail()`) so many times, now I'm going to ask:
are these neccessary? I think if there's a not expected exception it will fail anyway.

I mean this is OK, I'm just curious.

Reply:
You are right, it is not necessary. To be honest, the base of the upgrader tests are copy-pasted from the tests of a different extension.
By the way, as i experienced, the not expected exceptions produce different console output. This way only the message is printed, so that it is easier to find.